darts/tests/test_backtesting.py

```diff
@@ -162,11 +162,19 @@ def test_backtest_regression(self):
         features_multivariate = (gaussian_series + sine_series).stack(gaussian_series).stack(sine_series)
         target = sine_series
 
-        features = TimeSeries.from_dataframe(features.pd_dataframe().rename({"0": "Value0", "1": "Value1"}, axis=1))
-        features_multivariate = TimeSeries.from_dataframe(features_multivariate.pd_dataframe().rename(
-            {"0": "Value0", "1": "Value1", "2": "Value2"}, axis=1)
+        features = (
+            features
+            .with_column_renamed(features.components[0], "Value0")
+            .with_column_renamed(features.components[1], "Value1")
         )
 
+        features_multivariate = (
+            features_multivariate
+            .with_column_renamed(features_multivariate.components[0], "Value0")
```

Comment:
Would be cleaner to be able to call
`with_columns_renamed(features_multivariate.components, ["Value0", "Value1", "Value2"])` no? Could also be more efficient because every call copies the data over (as a new TimeSeries creation).

Reply:
see my comment below, I can change it